docs/docusaurus.config.js

```diff
@@ -65,16 +65,10 @@ module.exports = {
       items: [
         {
           type: "doc",
-          label: "Guides",
+          label: "Documentation",
```

Comment:
The terminology seems not very precise, but I don't have a better idea.

Reply:
Yes, it's all except api reference and self-hosting instructions. Alternatives I've thought of were "Home" or "Product" but was also not happy with it.